CHANGELOG-UNRELEASED.md

```diff
@@ -7,7 +7,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 * Adds EncryptedSeed and seed.encrypt() allow for easy passphrase encrypting/decrypting of any of the existing seed types. Adds the MnemonicableSeed trait allows seeds to be converted to/from BIP39 mnemonics. [#1687](https://github.com/holochain/holochain-rust/pull/1687) 
-- added nix for `hc-conductor-install` and `hc-conductor-uninstall` based on `cargo`
+* added nix for `hc-conductor-install` and `hc-conductor-uninstall` based on `cargo`
```

Comment:
looks like this got added back in incorrectly.

Reply:
I just changed the `-` to `*` for consistency